test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -88,7 +89,8 @@ def setup
         version: '2.0.0',
         eci: '06',
         cavv: 'AgAAAAAAAIR8CQrXcIhbQAAAAAA',
-        ds_transaction_id: 'MDAwMDAwMDAwMDAwMDAwMzIyNzY='
+        ds_transaction_id: 'MDAwMDAwMDAwMDAwMDAwMzIyNzY=',
+        authentication_response_status: 'Y'
```

Comment:
Out of pure morbid curiosity I tried these remote tests with `authentication_response_status: 'mwahahahaha'` and everything still passed. Not sure that really tells us much except that this value many not mean anything to Checkout's sandbox.

Reply:
Yep, this is more of a negative check on that field. Checkout doesn't provide us MADA cards to test against, otherwise we could write more tests in which adding this field would be more applicable.